deny.toml

```diff
@@ -7,14 +7,14 @@ confidence-threshold = 1.0
 # List of explictly allowed licenses
 # See https://spdx.org/licenses/ for list of possible licenses
 # [possible values: any SPDX 3.11 short identifier (+ optional exception)].
-allow = ["Unicode-3.0", "Apache-2.0", "BSD-3-Clause", "ISC", "MIT", "MPL-2.0"]
-
-exceptions = [
-    # Ring has a...complicated license. However, since it is at the core of a large number of rust
-    # projects, we are manually allowing the OpenSSL part of the license
-    { name = "ring", allow = [
-        "OpenSSL",
-    ] },
+allow = [
+    "Apache-2.0",
+    "BSD-3-Clause",
+    "CDLA-Permissive-2.0",
```

Comment:
Just adding a note here that this is a Linux Foundation driven license, so this should be totally ok